.travis.yml

```diff
@@ -1,4 +1,4 @@
-
+dist: trusty
```

Comment:
Could you change the commit message for this change to describe the problem. We have context at the moment, because we've changed a few of them, but it will be confusing to look back on in the future. You can copy/paste from alphagov/paas-cf@e7ff9a7786668334b41ce733d48c192716b39941.